lib/appointmentData.js

```diff
@@ -1,36 +1,36 @@
 import { nanoid } from "nanoid";
 
 export const appointmentData = [
-  {
-    id: nanoid(),
-    name: "Alex",
-    green: "Putz",
-    text: "Treppenhaus",
-    date: "13.12.2022",
-    clock: "14:00",
-  },
-  {
-    id: nanoid(),
-    name: "Simone",
-    green: "Putz",
-    text: "Treppenhaus",
-    date: "13.12.2022",
-    clock: "12:00",
-  },
-  {
-    id: nanoid(),
-    name: "Sebastian",
-    green: "Putz",
-    text: "Treppenhaus2",
-    date: "14.12.2022",
-    clock: "13:00",
-  },
-  {
-    id: nanoid(),
-    name: "Michael",
-    green: "Putz",
-    text: "Mülltonne",
-    date: "15.12.2022",
-    clock: "12:00",
-  },
+       id: nanoid(),
+     {
+       name: "Alex",
```

Comment:
bin nicht sicher, was hier passiert ist. wäre es nicht sinnvoller, die nanoid in deinem Alex-Objekt zu lassen? (vielleicht lese ich den Code aber auch gerade falsch...)

Reply:
Die Dummy-Daten muss ich demnächst löschen.